lib/hummingbird.ex

```diff
@@ -70,12 +70,15 @@ defmodule Hummingbird do
       data: %{
         conn: Helpers.sanitize(conn),
         caller: opts.caller,
-        trace_id: conn.assigns[:trace_id],
-        span_id: conn.assigns[:span_id],
-        parent_id: conn.assigns[:parent_id],
+        traceId: conn.assigns[:trace_id],
+        id: conn.assigns[:span_id],
+        parentId: conn.assigns[:parent_id],
         user_id: conn.assigns[:current_user][:user_id],
         route: conn.assigns[:request_path],
-        service_name: opts.service_name
+        serviceName: opts.service_name,
+        name: opts.service_name
```

Comment:
This is the only one we'll have to looksee if it needs to change.  :shakes-fist:

Reply:
I think this one is pretty safe. The istio ones look like `relay-web.default.svc.cluster.local:80/*` or `relaytms.com`